packages/admin/admin/src/tabs/Tabs.tsx

```diff
@@ -82,24 +125,34 @@ function TabsComponent({
                     const { children, label, ...restTabProps } = child.props;
                     return <TabComponent label={label} {...restTabProps} />;
                 })}
-            </MuiTabs>
+            </StyledTabs>
             {React.Children.map(children, (child: React.ReactElement<TabProps>, index) => {
+                const ownerState: OwnerState = {
+                    forceRender: child.props.forceRender,
```

Comment:
In the case that `index` and `value` are the same, the `display: none` styling was previously not applied, this should stay the same.  

```suggestion
                    contentHidden: index !== value && child.props.forceRender,
```

Reply:
should I change `forceRender` to `contentHidden?: boolean` in OwnerState?

Reply:
Yes, as it's not the same value as `props.forceRender` it should at least not be named `forceRender`